include/errno.h

```diff
@@ -51,6 +51,8 @@
  * returned by strerror().
  */
 
+/* usr/include/asm-generic/errno-base.h */
```

Comment:
```suggestion
/* linux/include/asm-generic/errno-base.h */
```
just for more clear reference

Reply:
Can it be /* Sync with linux/include/asm-generic/errno-base.h */
or /* Based on linux/include/asm-generic/errno-base.h */ ?

Reply:
sure